Domain.Supply/Entities/Supply/SupplyEntity.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Diagnosea.Submarine.Abstractions.Enums.Supply;
+
+namespace Diagnosea.Submarine.Domain.Supply.Entities.Supply
+{
+    public class SupplyEntity
+    {
+        public Guid Id { get; set; }
+        public Guid SupplyStoreId { get; set; }
+        public string Name { get; set; }
+        public SupplyCategory Category { get; set; }
+        public int OwnedQuantity { get; set; }
```

Comment:
Going to have to be more devisive about declaring owned quantities. 

* For food this may be like 50g, which we can deduct from when feedings happen. 
* For medicine this may be like 200ml, which we can deduct from when treatments happen.